File: LSP_ThirdWheel/LSP_ThirdWheel/CSharp/Square.cs
namespace CSharp
{
    public class Square : Rectangle
    {
        private int _sidelength;

        public override int Width
        {
            get { return _sidelength; }
            set { _sidelength = value; }
        }

        public override int Height
        {
            get { return _sidelength; }
            set { _sidelength = value; }
        }
    }
}

File: LSP_ThirdWheel/LSP_ThirdWheel/VisualBasic/Square.vb
Public Class Square
    Inherits Rectangle

    Private _sideLength As Integer

    Public Overrides Property Width As Integer
        Get
            Return _sideLength
        End Get
        Set(value As Integer)
            _sideLength = value
        End Set
    End Property

    Public Overrides Property Height As Integer
        Get
            Return _sideLength
        End Get
        Set(value As Integer)
            _sideLength = value
        End Set
    End Property
End Class